backend/in_memory_storage/in_memory_storage.go

```diff
@@ -31,10 +31,48 @@ type inMemStore struct {
 	settings models.Settings
 }
 
-func MakeInMemStore() storage.StorageService {
+func MakeInMemStore(defaultDomain string) storage.StorageService {
+	defaultPermissionList := models.Permission{Allow: []models.UserMatcher{{
+		Type: models.UserMatcherTypeDomain,
+		ID:   defaultDomain,
+	}}}
+	teamPermissions := models.TeamPermissions{
+		Read:  defaultPermissionList,
+		Write: defaultPermissionList,
+	}
+
+	// Sample users that are used in unit tests
```

Comment:
Ideally these users shouldn't be present when people are using the in-mem store for local manual testing and development. We should probably have another constructor, called by the unit tests, that adds them.

It's OK to go this way for now, but can you at least add a TODO to the code for this?

Reply:
I added a parameter in the constructor determining whether the test users should be added. It's probably not ideal but should be easy to refactor because the code for the test users is all separate now.